cartridges/openshift-origin-cartridge-haproxy/bin/control

```diff
@@ -175,17 +179,22 @@ function _reload_service() {
 
 function start() {
     _start_haproxy_service
-    isrunning  &&  echo "HAProxy instance is started"
+    isrunning  &&  client_result "HAProxy instance is started"
```

Comment:
I feel like this should be an `if` statement that prints a warning if `isrunning` fails, but that may be beyond scope for this PR.